teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
```

Comment:
- [ ] Same requirement as mentioned above, kindly ensure the parameters are initialized in the correct order.

Reply:
Thanks @zhadier